Introduction/Getting started/Hello World/task.md

```diff
@@ -17,7 +17,7 @@ Now let's replace the line inside the main function with the code:
 ```
 ##### Example: A program that prints Hello, world!
 
-To run this code click on **Run** icon at the left-upper corner of the editor. Click **Check** button in the **Task Description** window to run the tests checking the correctness of the solution provided.
+To run this code click on **Run** icon (![](images/run.svg)) at the left-upper corner of the editor. Click **Check** button in the **Task Description** window to run the tests checking the correctness of the solution provided.
```

Comment:
To run this code, click the **Run** icon (![](images/run.svg)) in the top-left corner of the editor. Click the **Check** button in the **Task Description** window to run tests and verify the correctness of your solution.